Kajabi/sage-lib
# Adjustments to toast functionality

## Description
Updates to toast display to be triggered by clicking on an element (button) within docs site.
In addition, allows toast to be displayed automatically by triggering an event.

### Screenshots
Current visuals do not change with these updates.


### Steps for testing
1. Visit http://localhost:4000/pages/breakout/element/toast
2. Click on the buttons and verify toast appears.
